src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala

```diff
@@ -19,25 +19,25 @@ object HTTP extends Logging {
     System.exit(0)
   }
   implicit private val materializer = ActorMaterializer()
-  val wsClient = StandaloneAhcWSClient()
 
   def getXml(lastid: Long): Either[LambdaError, String] = Switches.rcsEnabled {
     logger.info(s"Fetching XML from $rcsUrl?lastid=$lastid&subscribername=$subscriberName")
+    val wsClient = StandaloneAhcWSClient()
     try {
       Await.result(wsClient.url(rcsUrl).withQueryStringParameters(("lastid", lastid.toString), ("subscribername", subscriberName)).get().map { result =>
         logger.info(s"Status of GET request was ${result.status}")
         result.status match {
           case 200 => Right(result.body)
           case _ => Left(RCSError(result.status + ": " + result.body))
         }
-      }, 10.minutes)
+      }, 2.minutes)
```

Comment:
does this need to relate to the [lambda timeout](https://github.com/guardian/rcs-poller-lambda/blob/master/cfn.yaml#L149)?

Reply:
It leads to a longer delay before detection of a failure.  I sincerely hope that we are not doing a call to check connectivity which takes the server ten minutes to process..?

Reply:
We're talking to a web server hosted within an Oracle database... who knows how long it'll take...